Makefile

```diff
@@ -158,7 +158,7 @@ WEBHOOK_ROOT ?= $(MANIFEST_ROOT)/webhook
 RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
 ASO_CRDS_PATH := $(MANIFEST_ROOT)/aso/crds.yaml
 ASO_VERSION := v2.3.0
-ASO_CRDS := resourcegroups.resources.azure.com
+ASO_CRDS := resourcegroups.resources.azure.com natgateways.network.azure.com
```

Comment:
@nojnhuh is this list going to keep growing everytime we add a new CRD? would it make sense to track it in some other way once the list ends up being more than a few?

Reply:
Yes, this will eventually get unwieldy here. Didn't get around to finding a better mechanism yet.

Reply:
not blocking for this PR, we can circle back once the list gets big enough